packages/video_player/video_player_platform_interface/lib/video_player_platform_interface.dart

```diff
@@ -100,7 +100,7 @@ abstract class VideoPlayerPlatform extends PlatformInterface {
   }
 
   /// Sets the audio mode to mix with other sources
-  Future<void> setMixWithOthers(bool mixWithOthers) {
+  Future<void> setMixWithOthers(bool mixWithOthers, bool ambient) {
```

Comment:
This is a breaking change; please see https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changing-platform-interface-method-parameters